commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceInventoryTestUtil.java

```diff
@@ -43,6 +43,28 @@ public static CommerceCountry addCommerceCountry(
 			RandomTestUtil.nextInt(), false, 0, true, serviceContext);
 	}
 
+	public static CommerceInventoryWarehouse addCommerceInventoryWarehouse()
+		throws PortalException {
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext();
+
+		CommerceCountry commerceCountry = _setUpCountry(serviceContext);
+
+		CommerceRegion commerceRegion = _setUpRegion(
+			commerceCountry, serviceContext);
+
+		return CommerceInventoryWarehouseLocalServiceUtil.
+			addCommerceInventoryWarehouse(
+				RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+				true, RandomTestUtil.randomString(),
+				RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+				RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+				commerceRegion.getCode(),
+				commerceCountry.getTwoLettersISOCode(), 45.4386111, 12.3266667,
```

Comment:
What are these `45.4386111` and `12.3266667` values? Could they be replaced with `RandomTestUtil.randomDouble()`? That way it's documented that the values don't matter.